package.json

```diff
@@ -11,6 +11,7 @@
   "main": "build/src/index.js",
   "typings": "build/src/index.d.ts",
   "scripts": {
+    "prepare": "npm run dist",
```

Comment:
I'm not sure we want this. Even if we do, this isn't the PR for it, so please remove.

Reply:
removed